views/list.hbs

```diff
@@ -20,5 +21,7 @@
       </li>
       {{/each}}
     </ul>
+    {{/if}}
+    <p>Click <a href="./">here</a> to start a new playground</p>
```

Comment:
Can't we use `href="/"` here?
`./` is weird, plus I think it would work from `http://paysage.xyz/list` but not `http://paysage.xyz/list/`.